cmd/celestia/bridge.go

```diff
@@ -1,4 +1,3 @@
-//nolint:dupl
```

Comment:
Are you sure this should be removed?

Reply:
Yeah. Linter was bugging us with duplicate code while it was not really so we have to put nolint there. And this change finally allows us to remove nolint